commands/conduit_cli.py

```diff
@@ -35,14 +35,14 @@ def get_cluster_arn(app: str, env: str) -> str:
             return cluster_arn
 
 
-def get_postgres_secret(app: str, env: str):
-    secret_name = f"/copilot/{app}/{env}/secrets/POSTGRES"
+def get_postgres_secret(app: str, env: str, name: str = "POSTGRES"):
```

Comment:
Does it need to be defaulted here as well as in the create_task?

Reply:
I've removed the defaulting entirely, allowing click to specify the default at the top level.